### ISSUE TITLE
Allow accessibility audit to pass in scheduled CI

### ISSUE THREAD

**Issue by ijlee2** [ijlee2/ember-container-query#83]
## Describe the bug 🐞

The CI, scheduled on May 3, 2021, didn't pass due to an accessibility issue in the demo app. Study the [provided documentation](https://dequeuniversity.com/rules/axe/4.2/aria-allowed-attr?application=axeAPI), then fix the issue.

```
not ok 1 Chrome 90.0 - [565 ms] - Acceptance | album: @w1 @h3 Accessibility audit
  ---
    actual: >
      null

    stack: >
      Error: The page should have no accessibility violations. Violations:
      [critical]: Elements must only use allowed ARIA attributes 
      Violated 5 times. Offending nodes are: 
      <span data-test-field="Explicit" aria-label="Explicit">
      <span data-test-field="Explicit" aria-label="Explicit">
      <span data-test-field="Explicit" aria-label="Explicit">
      <span data-test-field="Explicit" aria-label="Explicit">
      <span data-test-field="Explicit" aria-label="Explicit">
      https://dequeuniversity.com/rules/axe/4.2/aria-allowed-attr?application=axeAPI
      To rerun this specific failure, use the following query params: &testId=bd497914&enableA11yAudit=true
        at DEFAULT_REPORTER (http://localhost:7363/assets/test-support.js:8445:13)
        at async Object.<anonymous> (http://localhost:7363/assets/tests.js:41:7)

    message: >
      Promise rejected during "@w1 @h3 Accessibility audit": The page should have no accessibility violations. Violations:
      [critical]: Elements must only use allowed ARIA attributes 
      Violated 5 times. Offending nodes are: 
      <span data-test-field="Explicit" aria-label="Explicit">
      <span data-test-field="Explicit" aria-label="Explicit">
      <span data-test-field="Explicit" aria-label="Explicit">
      <span data-test-field="Explicit" aria-label="Explicit">
      <span data-test-field="Explicit" aria-label="Explicit">
      https://dequeuniversity.com/rules/axe/4.2/aria-allowed-attr?application=axeAPI
      To rerun this specific failure, use the following query params: &testId=bd497914&enableA11yAudit=true

    negative: >
      false

    browser log: |
  ...
```

## Expected behavior 🤔

CI should pass.

## Minimal reproduction 🔬

N/A

## Environment 🌍

N/A

## Additional context ➕

N/A
